ci/redhat-ci.sh

```diff
@@ -36,12 +36,9 @@ install_dependencies() {
 
     # install apisix-runtime to make apisix's rpm test work
     yum install -y yum-utils && yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
-    rpm --import https://repos.apiseven.com/KEYS
-    yum -y install https://repos.apiseven.com/packages/centos/apache-apisix-repo-1.0-1.noarch.rpm
-
-    export luajit_xcflags="-DLUAJIT_ASSERT -DLUAJIT_NUMMODE=2 -DLUAJIT_ENABLE_LUA52COMPAT -O0"
-    export debug_args=--with-debug
+    yum install -y openresty-pcre-devel openresty-zlib-devel
```

Comment:
why add this?

Reply:
It is required to build the apisix-runtime.